osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/ConfigurationManagementServiceTest.java

```diff
@@ -0,0 +1,290 @@
+package org.opensmartgridplatform.adapter.domain.core.application.services;
```

Comment:
whole file: Add some blank lines to arrange some code blocks.
See https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html

Reply:
you've created the fields
    private final String organisationIdentification = "a";
    private final String deviceIdentification = "b";
    private final String correlationUid = "c"
and still uses "a", "b" and "c" in the test code. This is very odd.
Refactor the fields to constants, and use these constants in the test code.

Reply:
reduce duplicate code by introducing some methods

Reply:
I removed some unnecessary parts, but which duplicate code needs to be reduced. I only have blocks of Mockito.when and Assertions.assertThat. These blocks very between classes and don't seem very suitable for being placed in a seperate method.

Reply:
For example, some when blocks could be refactored to a method
{code}
    private void arrangeOrganisationDeviceAndIpAddress(Organisation organisation, Device activeDevice, String ipAddress)
            throws UnknownEntityException, FunctionalException {
        when(this.organisationDomainService.searchOrganisation(any(String.class))).thenReturn(organisation);
        when(this.deviceDomainService.searchActiveDevice(any(), any())).thenReturn(activeDevice);
        when(activeDevice.getIpAddress()).thenReturn(ipAddress);
    }
{code}